evm_loader/program/src/executor_state.rs

```diff
@@ -166,6 +171,7 @@ pub struct ExecutorSubstate {
     spl_approves: Vec<SplApprove>,
     erc20_allowances: BTreeMap<(H160, H160, H160, Pubkey), U256>,
     deletes: BTreeSet<H160>,
+    query_account_cache: query::AccountCache,
```

Comment:
We change ExecutorSubstate, so we should update Storage::TAG to the new version.
Otherwise, we may get problems with those transactions that were executed at the time of the update (we can't parse it storage properly).

Reply:
We have this set of tags:

```
    const EMPTY_TAG: u8 = 0;
    const ACCOUNT_TAG: u8 = 1;
    const CONTRACT_TAG: u8 = 2;
    const STORAGE_TAG: u8 = 3;
    const ERC20_ALLOWANCE_TAG: u8 = 4;
```

Should the STORAGE_TAG be changed to 5, for example?